isovar/cli/variant_sequences.py

```diff
@@ -31,24 +32,31 @@
 def add_variant_sequence_args(
         parser,
         add_sequence_length_arg=False):
-    rna_sequence_group = parser.add_argument_group("Consensus cDNA sequence")
+    rna_sequence_group = parser.add_argument_group(
+        "Determine coding sequence from RNA")
+
+    rna_sequence_group.add_argument(
+        "--min-alt-rna-reads",
+        default=MIN_ALT_RNA_READS,
+        help="Minimum number of reads supporting variant allele (default %(default)s)")
+
     rna_sequence_group.add_argument(
-        "--min-reads-supporting-variant-sequence",
+        "--min-variant-sequence-coverage",
         type=int,
-        default=MIN_READS_SUPPORTING_VARIANT_CDNA_SEQUENCE,
+        default=MIN_VARIANT_SEQUENCE_COVERAGE,
         help="Minimum number of reads supporting a variant sequence (default %(default)s)")
 
     rna_sequence_group.add_argument(
-        "--variant-sequence-assembly",
-        default=False,
-        action="store_true")
+        "--disable-variant-sequence-assembly",
```

Comment:
why not default to `--variant-sequence-assembly True` in that case? then the user can disable it if they need to